datafusion/expr/src/logical_plan/mod.rs

```diff
@@ -20,6 +20,8 @@ mod ddl;
 pub mod display;
 pub mod dml;
 mod extension;
+pub(crate) mod invariants;
```

Comment:
I didn't see a new `invariants` module in this PR 🤔

Reply:
🤦🏼 . Fixed. TY.